ipam/client/backends/phpipam.py

```diff
@@ -286,18 +289,14 @@ def get_num_ips_by_desc(self, description):
                               AND state = '1'"
                          % (description))
         row = self.cur.fetchone()
-        if row is not None:
-            return int(row[0])
-        return None
+        return int(row[0])
 
     def get_num_subnets_by_desc(self, description):
         self.cur.execute("SELECT COUNT(subnet) FROM subnets \
                          WHERE description LIKE '%s'"
                          % description)
         row = self.cur.fetchone()
-        if row is not None:
-            return int(row[0])
-        return None
+        return int(row[0])
```

Comment:
How is handled the TypeError if row is None?

Reply:
There is no chance to have this case as I do a count operation. Tests never triggered this code path, while testing edge cases with 0 results (but still a row line containing 0).